hw/ip/soc_dbg_ctrl/data/soc_dbg_ctrl_testplan.hjson

```diff
@@ -3,37 +3,31 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name: "soc_dbg_ctrl"
-  // TODO: remove the common testplans if not applicable
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
   testpoints: [
     {
       name: smoke
       desc: '''
-            Smoke test accessing a major datapath within the soc_dbg_ctrl.
+            Verify the soc_dbg_ctrl policy bus and decoding logic.
 
-            **Stimulus**:
-            - TBD
+            Stimulus:
+              - Let the soc_dbg_state_i be in SocDbgStProd
+              - Write a random valid debug category to DEBUG_POLICY_CATEGORY_SHADOWED
+              - Randomize DEBUG_POLICY_RELOCKED between Mubi4::True and Mubi4::False
+              - Write DEBUG_POLICY_VALID_SHADOWED to Mubi4::True
 
-            **Checks**:
-            - TBD
+            Checking:
+              - The attacked soc_dbg_ctrl_decode module decodes must decode the debug policy bus
```

Comment:
```suggestion
              - The instantiated soc_dbg_ctrl_decode module decodes must decode the debug policy bus
```

Reply:
Wouldn't it make more sense to check `soc_dbg_policy_bus_o` of `soc_dbg_ctrl`, though?